2024-03-03 11:48:05.288592
<class 'AttributeError'>
AttributeError("'Tensor' object has no attribute 'astype'")
  File "D:\XDU_Pattern_Recgonition\KNN\KNN_Usps_GPU.py", line 83, in <module>
    Res[m] = k_nn(train_data_tensor,train_label_tensor,test_data_tensor,test_label_tensor)

  File "D:\XDU_Pattern_Recgonition\KNN\KNN_Usps_GPU.py", line 65, in k_nn
    a = a.astype(int)


